src/Dropdown/Dropdown.stories.js

```diff
@@ -49,6 +87,14 @@ Default.args = {
   placeholder: 'Select Day',
 }
 
+export const DefaultWithGroups = Template.bind({})
+
+DefaultWithGroups.args = {
+  id: 'days',
+  list: daysWithGroups,
```

Comment:
Maybe better to keep the default one without groups?

Reply:
@fleonard We've got both still:

```javascript
export const Default = Template.bind({})

Default.args = {
  id: 'days',
  list: days,
  placeholder: 'Select Day',
}

export const DefaultWithGroups = Template.bind({})

DefaultWithGroups.args = {
  id: 'days',
  list: daysWithGroups,
  placeholder: 'Select Day',
}
```

Reply:
missed it